docs/docs/recipes/deployment/README.md

```diff
@@ -90,4 +90,6 @@ You are all set. Open the browser and visit your domain URL, you should be able
 
 If we don't mention it in the changelog, you can feel free to upgrade Logto without changing your code or database schema. Our API follows [semver](https://semver.org/).
 
-We'll build a smooth database migration process and demonstrate it in our changelog if a schema change is inevitable. You'll be able to do that with chill.
+### Database migration
```

Comment:
should be "alternation"? then we can separate from "migrate from other projects"

Reply:
Sure, then I'll also change `migration` from code later.